src/debug/di/cordb.cpp

```diff
@@ -499,7 +506,7 @@ DbiGetThreadContext(HANDLE hThread,
     DT_CONTEXT *lpContext)
 {
     // if we aren't local debugging this isn't going to work
-#if !defined(_ARM_) || defined(FEATURE_DBGIPC_TRANSPORT_DI) || defined(__ANDROID__)
+#if !defined(_ARM_) || defined(FEATURE_DBGIPC_TRANSPORT_DI) || !HAVE_ALIGNED_MALLOC
```

Comment:
Not sure. All I know is that when the unix transport support was added, it was really only not enabled for Windows.  For the platforms that can't support it or it just doesn't make sense (like on a phone), it is ok.  As a way to disable this transport for platforms that already support it, I don't have an opinion.